torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -5092,7 +5092,7 @@ def test_ddp_logging_data_cpu(self):
             def parse_env(var):
                 return os.environ[var] if var in os.environ else "N/A"
 
-            os.environ["TORCH_DISTRIBUTED_DEBUG"] = "INFO"
+            dist.set_debug_level(dist.DebugLevel.INFO)
```

Comment:
do we need a test for `dist.set_debug_level(dist.DebugLevel.DETAIL)` to ensure that works properly? I'm pretty sure there should be a test somwhere here that does `TORCH_DIST_DEBUG=DETAIL`, and ensures that a wrapper process group is created, we can have a similar test if you think that would be beneficial.

Reply:
Yes that makes sense. I don't have much context about what assertions such test should include though. It would be great if someone more knowledgeable could work on that.

Reply:
I can add that later